docs/apache-airflow-providers-fab/img/diagram_fab_auth_manager_airflow_architecture.py

```diff
@@ -0,0 +1,73 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
You mixed the two diagrams. This one goes to Airflow doc and the other goes here in the fab provider (look at the images, they are not the same)

Reply:
🤦

Reply:
Solved.